src/app/routes/admin/admin.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="container-fluid">
+  <table *ngIf="admin"  class="table table-sm table-striped mt-3">
```

Comment:
This table looks great! I think we should have the title link go to the edit page . **Also you should use angular router links not href tags.** The href tags usually trigger an entire page reload so it's best to use them only when you're redirecting to a page that isn't in the same Single Page App.  

You can check out the revisions link on the admin edit page to see an example of a router link.

Reply:
It might also be nice to have a title or something that says. "Admin" with a subtitle of "Pages you can edit". Or whatever makes sense to give the page some context.